firmware/ci.c

```diff
@@ -959,6 +960,24 @@ void ci_service(void)
 
 	token = get_token(&str);
 
+	if(strcmp(token, "") == 0) {
+		if(blank_command_counter == 1) {
```

Comment:
(Have changed it to init as 2 instead, so this now works as a countdown with the int decrementer.)